### DISCUSSION TITLE
New release?

### DISCUSSION THREAD

**Discussion by marcelotrevisani** [topgrade-rs/topgrade#917] [General]
Hey, folks do you have a plan when you are going to release the next version please?

**Comment by lucaspar** [comment]
as a follow up, is there a scheduled release cycle topgrade follows?

**Comment by SteveLauC** [comment]
Hi, I plan to release the new version in the next week.

> as a follow up, is there a scheduled release cycle topgrade follows?

Currently no🤦, maybe we should do regular releases

**Comment by GideonBear** [comment]
Ref #1141
